net/test/test.sh

```diff
@@ -45,7 +45,7 @@ tailseek=$(( $(stat -c %s "$log") + 1 ))
 
 # currently, kernel UT runs as part of loading m0ut module
 modload
-insmod $MODMAIN $*
+insmod $MODMAIN "$*"
```

Comment:
we don't have any kernel modules now.. expect in case of lnet.
do we need this change?

Reply:
This is wrong.
`$*` means, all arguments. They will be passed to `insmod` respectively.
But `"$*"` will combine all arguments into a single argument.

If you run `sh net/test/test.sh first.ko second.ko third.ko`, it will combine the three arguments into a single one.
That's wrong.

I have reviewed similar changes, and they all have the same mistake. Please correct them.